src/components/Sidebar/SidebarView.js

```diff
@@ -5,6 +5,7 @@ import Tab from '@material-ui/core/Tab';
 import { withStyles } from '@material-ui/core/styles';
 import SearchContainer from './SearchContainer';
 import SchoolsContainer from './SchoolsContainer';
+import ConnectedCartContainer from './CartContainer';
```

Comment:
`Connected` in the name is redundant since in our repo, any `Container` component means it is connecting its child to the redux store.

Reply:
This was a remnant of ESLint's request back when I had the container coded as a full-fleged `Component`--I changed this as well.